tools/run_tests/run_interop_tests.py

```diff
@@ -1519,9 +1519,6 @@ def aggregate_http2_results(stdout):
         sys.exit(1)
     else:
         sys.exit(0)
-except Exception as e:
```

Comment:
this was originally added here: https://github.com/grpc/grpc/pull/9776/files#diff-267afa22a65534e2688adc5fce0f3f47R1151
I'm pretty sure we don't want that because it can easily mask script crashing as "green" tests (as we saw)